packages/features/bookings/Booker/components/Header.tsx

```diff
@@ -43,7 +42,7 @@ export function Header({
     [setLayout, layout]
   );
 
-  if (isMobile || isEmbed || !enabledLayouts) return null;
+  if (isMobile || !enabledLayouts) return null;
```

Comment:
We don't want to hide the entire header in embed. We just want the layout toggle to go away for embed.